lib/internal/mime.js

```diff
@@ -270,7 +291,7 @@ class MIMEParams {
                                   NOT_HTTP_TOKEN_CODE_POINT) === -1 &&
         SafeStringPrototypeSearch(parameterValue,
                                   NOT_HTTP_QUOTED_STRING_CODE_POINT) === -1 &&
-        params.has(parameterString) === false
+        paramsMap.has(parameterString) === false
```

Comment:
I think here was a bug, as params does not have a method called .has() but I guess we lack here some code coverage.

Reply:
It has it, doesn't it? https://github.com/nodejs/node/pull/49889/files#diff-b95a1c4e34435c915f5385c9a8f3f89ebfee8aa994049606400c5fe163eb523dL140